docs/image-sizes.md

```diff
@@ -12,6 +12,16 @@ They have default dimensions, but you can override these either with code or in
 
 ![Media Settings](./assets/media-settings.png)
 
+### `content_width`
+
+If your theme defines a content width via the `$content_width` (or `$GLOBALS['content_width]`) global, this size will _override_ the size defined on the Settings > Media page. This will result in the Large size appearing to be the "wrong" size (using the size defined in the _global_ rather than the size in the admin) in the media modal.
```

Comment:
We can describe how it behaves here, this talks about overriding but really it's setting an upper limit on the width of images according to your theme's design.

The content width value should generally match the maximum width of your post's body text in pixels.